h2o-py/h2o/explanation/_explain.py

```diff
@@ -1598,7 +1598,7 @@ def varimp(
         :returns: either pandas DataFrame (if use_pandas == True) or a triple (varimps, model_ids, variable_names)
     """
     if _is_automl_or_leaderboard(models):
-        models = _get_models_from_automl_or_leaderboard(models, filter_=_has_varimp)
+        models = list(_get_models_from_automl_or_leaderboard(models, filter_=_has_varimp))
```

Comment:
why do you need a list? 
most Py operations accept an iterator: see my other comment about using `filter` (which returns an iterator) instead of generator.
I don't know if all models are always needed here.
If yes, then you can as well fetch them all from the start.
If no, then the iterator will ensure that further operations will fetch models only if needed.

Reply:
This uses all the models with varimp. All this function does is getting varimp from them, transform it so it contains varimp per variable (not the encoded variable) and it makes a matrix / pandas.dataframe if `use_pandas==True`.